astropy/utils/exceptions.py

```diff
@@ -78,3 +73,26 @@ def __repr__(self):
 
 
 NoValue = _NoValue()
+
+
+def __getattr__(name: str):
+    if name in ("ErfaError", "ErfaWarning"):
+        import warnings
+
+        warnings.warn(
+            f"Importing {name} from astropy.utils.exceptions was deprecated "
+            "in version 6.1 and will stop working in a future version. "
+            f"Instead, please use\nfrom erfa import {name}\n\n",
+            category=AstropyDeprecationWarning,
+            stacklevel=1,
+        )
+
+        if name == "ErfaError":
```

Comment:
I think this can just be
```
import erfa
return getattr(erfa, name)
```

Reply:
thank you ! I was initially going to use `importlib` to do this dynamically but thought a dumb implementation was just as good, I missed that I could use `getattr` (ironic given that I was implementing `__getattr__`...)